docs/integrations/saas-cloud/docusign.md

```diff
@@ -0,0 +1,206 @@
+| fillmissing timeslice,values all in severity
+| transpose row _timeslice column severity as low, medium, high
+```
+
+## Installing the DocuSign App
+
+Locate and install the app from the **App Catalog**. To preview the dashboards included with the app before installing, click **Preview Dashboards**.
+
+Before you begin, collect logs from DocuSign API and ingest them into Sumo Logic. Refer to the [DocuSign API Cloud-to-Cloud Integration](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source/) to create the source and use the same source category while installing the app.
+
+To install the app, follow the steps below:
+1. From the **App Catalog**, search for the app and select it.
+1. Select **Add Integration** button to install the app.
+1. Configure **DocuSign App** using the steps described in the [DocuSign Cloud-to-Cloud Source](docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source/). If you already have set up your data, skip this step by clicking on **Next**.
+1. Complete the following fields:
+   1. **Data Source**. Select either of these options for the data source:
+      * Choose **Source Category** and then choose a source category from the list.
+      * Select **Enter a Custom Data Filter** and type in a custom source category that starts with an underscore. For example, `_sourceCategory=MyCategory`.
+    2. **Folder Name**. You can retain the existing name, or enter a name of your choice for the app. 
+    3. Select the **Location in Library** (the default is the **Personal** folder in the library), or click **New Folder** to add a new folder.
+1. Click **Next**.
+
+Once an app is installed, it will appear in your **Personal** folder, or other folder that you specified. You can share it with your organization.
+
+The panels will begin to fill automatically. It's worth noting that each panel gradually fills with data that matches the time range query and has been received since the panel was created. The results will not be available right away, but with some patience, you will be able to view full graphs and maps.
+
+## Viewing the DocuSign Dashboards
+
+All dashboards have a set of filters that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard.
+
+You can use filters to drill down and examine the data on a granular level. Filters include client country, client device type, client IP, client request host, client request URI, client request user agent, edge response status, origin IP, and origin response status.
+
+Each panel has a set of filters that are applied to the results for that panel only, as shown in the following example. Click the funnel icon in the top panel menu bar to display a list of panel-specific filters.
+
+### DocuSign - Overview Dashboard
```

Comment:
We don't use "Dashboard" in these H3 headers since it's already indicated in the parent H2 header (it'd look cluttered in the sidebar)
```suggestion
### DocuSign - Overview
```